include/RAJA/policy/cuda/launch.hpp

```diff
@@ -151,9 +153,11 @@ struct LaunchExecute<RAJA::policy::cuda::cuda_launch_explicit_t<async, named_usa
 
       RAJA_FT_BEGIN;
 
+      size_t shared_mem_size = launch_params.shared_mem_size;
       RAJA::cuda::detail::cudaInfo launch_info;
       launch_info.gridDim = gridSize;
       launch_info.blockDim = blockSize;
+      launch_info.dynamic_smem = &shared_mem_size;
```

Comment:
Why not make a copy? The potential for a dangling pointer is pretty high here.

Reply:
I have to modify the shmem used to launch the kernel.